### ISSUE TITLE
Cursor::getFirstNonSpacePosition() deprecated since 0.15.4

### ISSUE THREAD

**Issue by fisharebest** [thephpleague/commonmark-ext-table#12]
Starting with CommonMark 0.15.4, the function `Cursor::getFirstNonSpacePosition()` triggers a deprecation notice.

See https://github.com/thephpleague/commonmark/issues/280

```
#0 unknown:unknown ErrorException("/var/www/vendor/league/commonmark/src/Cursor.php:79 Cursor::getFirstNonSpacePosition() will be removed in a future 0.x release. Use getNextNonSpacePosition() instead. See https://github.com/thephpleague/commonmark/issues/280")
#1 /vendor/league/commonmark/src/Cursor.php:79 trigger_error('Cursor::getFirstNonSpacePositi…', 16384)
#2 /vendor/webuni/commonmark-table-extension/src/TableParser.php:40 getFirstNonSpacePosition()
#3 /vendor/league/commonmark/src/DocParser.php:192 parse([Context], [Cursor])
#4 /vendor/league/commonmark/src/DocParser.php:107 parseBlocks([Context], [Cursor])
#5 /vendor/league/commonmark/src/DocParser.php:83 incorporateLine([Context])
#6 /vendor/league/commonmark/src/Converter.php:59 parse('Image downloaded from 
```

**Comment by hason** [comment]
Fixed in 0.7 version.

**Comment by fisharebest** [comment]
Thanks!
